playground/html/__tests__/html.spec.ts

```diff
@@ -298,7 +298,7 @@ describe.runIf(isServe)('invalid', () => {
   })
 
   test('should close overlay when escape key is pressed', async () => {
-    await page.goto(viteTestUrl + '/invalid.html')
+    await page.goto(viteTestUrl + '/invalidEscape.html')
```

Comment:
The same error in the same file will not be printed repeatedly. To ensure that the test case is normal, multiple HTML files with the same content but different names are created.